src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin star-size {
+  width: 16px;
+  height: 16px;
+}
```

Comment:
looks like placeholder

let's pass some variable(with needed size) to make this mixin more useful